raiden/api/rest.py

```diff
@@ -92,15 +94,25 @@
     ('/tokens/<hexaddress:token_address>/partners', PartnersResourceByTokenAddress),
     ('/tokens/<hexaddress:token_address>', RegisterTokenResource),
     ('/events/network', NetworkEventsResource),
-    ('/events/tokens/<hexaddress:token_address>', TokenEventsResource),
+    ('/events/tokens/blockchain/<hexaddress:token_address>', TokenBlockchainEventsResource),
+    ('/events/tokens/raiden/<hexaddress:token_address>', TokenRaidenEventsResource),
     (
-        '/events/channels/<hexaddress:token_address>',
-        ChannelEventsResource,
-        'tokenchanneleventsresource',
+        '/events/channels/blockchain/<hexaddress:token_address>',
```

Comment:
use `hexaddress:channel_address` instead of `hexaddress:token_address`

Reply:
wait, from the endpoints bellow, this look intentional, so this PR currently has the following endpoints:

```
/events/tokens/blockchain/:token_address
/events/tokens/raiden/:token_address
/events/channels/blockchain/:token_address
/events/channels/raiden/:token_address
/events/channels/blockchain/:token_address/:partner_address
/events/channels/raiden/:token_address/:partner_address
```

IMO this structure is not clear, perhaps grouping the events on a top-level document:

```
/blockchain_events/tokens/:token_address
/blockchain_events/channels/:token_address
/blockchain_events/channels/:token_address/:partner_address
/raiden_events/tokens/:token_address
/raiden_events/channels/:token_address
/raiden_events/channels/:token_address/:partner_address
```

And adding the channel events bellow a token network document:

```
/blockchain_events/tokens/:token_address
/blockchain_events/network/:token_address/channels/
/blockchain_events/network/:token_address/channels/:partner_address
/raiden_events/tokens/:token_address
/raiden_events/network/:token_address/channels/
/raiden_events/network/:token_address/channels/:partner_address
```

Reply:
> **use hexaddress:channel_address instead of hexaddress:token_address**

We should have two endpoints as per the [issue](https://github.com/raiden-network/raiden/issues/1898) one for all channels of the token and then filter by channel which is filtered by token_address/partner_address

> And adding the channel events bellow a token network document:

Okay agreed. Will append to the issue so that @CosminNechifor knows what he has to do.